.github/workflows/java-build-for-release.yml

```diff
@@ -0,0 +1,92 @@
+name: Build Java Release
+on:
+  push:
+    tags:        
+      # if you change this pattern, make sure jobs.strip-tag still works
+      - 'release/java/v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  ci:
+    uses: ./.github/workflows/java-build.yml
+
+  strip-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+    steps:
+      - name: process tag 
+        id: version
+        run: |
+          echo "version=$(echo ${{ github.ref_name }} | cut -c15-)" >> $GITHUB_OUTPUT
+          
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [ci, strip-tag]
+    outputs:
+      hashes: ${{ steps.hash.outputs.hashes }}
+    steps:
+      - name: checkout tag
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        with:
+          ref: "${{ github.ref }}"
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@de1bb2b0c5634f0fc4438d7aa9944e68f9bf86cc # v3.6.0
+        with:
+          java-version: 8
+          distribution: 'temurin'
+
+      - name: Build project
+        run: |
+          # override the version in gradle.properties
```

Comment:
Can you elaborate a little more on why the `java-build.yml` call isn't sufficient here?

It's because (1) you need to stick `version` in there, and (2) `createReleaseBundle` is a different target from `build`?

Reply:
Yeah injecting the version is the main part. The other option is to check in the version on every update to settings.gradle.kts (which I don't like in a repo like this). 

createReleaseBundle is a custom task which generates output in a way that makes it easier to use the slsa-generator AND sign and release on another machine. I'm not tied to this task, but it's the most familiar mechanism to me for now.